src-v2/Judge/NTS.Judge/Adapters/Behinds/DashboardBehind.cs

```diff
@@ -63,6 +84,68 @@ async Task CreateOfficials(IEnumerable<Domain.Setup.Entities.Official> setupOffi
         }
     }
 
+    async Task CreateParticipations(Domain.Setup.Entities.Event setupEvent) 
+    {
+        foreach(var competition in setupEvent.Competitions)
+        {
+            var setupPhases = competition.Phases;
+            var competitionDistance = 0m;
+            var phases = new List<Phase>();
+            var categoryEntriesPairs = new Dictionary<AthleteCategory, List<RankingEntry>>
```

Comment:
This isn't a very good dictionary name:
1. What entries
2. What pairs

A good naming for dictionaries is $"{values}By{Category}", aka: rankingEntriesByCategory.